tetrateci/port_patches_to_release.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+
+if [[ -z "$GITHUB_TOKEN" ]]; then
+  echo "Undefined GITHUB_TOKEN environment variable."
+  exit 1
+fi
+
+echo "Configuring git"
+
+cat <<- EOF > $HOME/.netrc
+    machine github.com
+    login $GITHUB_ACTOR
+    password $GITHUB_TOKEN
+    machine api.github.com
+    login $GITHUB_ACTOR
+    password $GITHUB_TOKEN
+EOF
+chmod 600 $HOME/.netrc
+
+git config user.name $GITHUB_ACTOR
+git config user.email github-actions@github.com
+
+echo "Fetching target branches"
+
+TARGETS=$(git branch -r| grep origin/tetrate-release | xargs)
+
+echo "Creating PRs"
+
+for branch in $TARGETS; do
+    echo "Getting branch name for $branch"
+    branch_name=$(cut -f2 -d"/" <<< $branch)
+    echo "Creating PR for $branch_name"
+    hub pull-request -b $branch_name -m "AUTO: Backporting patches to $branch_name"
```

Comment:
Who would be the reviewers? Do we need to create a group and include by default?

Reply:
I haven't assigned any reviewers yet, we can use a set of handles for users or a team with the `-r` flag to set that